karma.conf.js

```diff
@@ -54,26 +54,30 @@ module.exports = function (config) {
           functions: 80,
           lines: 80,
           excludes: [
+            'src/actions/transactions.js',
+            'src/components/errorBoundary/index.js',
+            'src/components/feedbackForm/**/*.js',
+            'src/components/ledger/accountCard.js',
+            'src/components/ledger/addAccountCard.js',
+            'src/components/ledger/index.js',
+            'src/components/ledger/ledgerLogin.js',
+            'src/components/ledger/unlockWallet.js',
+            'src/components/login/index.js',
+            'src/components/login/login.js',
             'src/components/register/register.js',
+            'src/components/request/specifyRequest.test.js',
             'src/components/voteUrlProcessor/index.js',
+            'src/store/middlewares/login.js',
+            'src/store/reducers/liskService.js',
+            'src/utils/api/ledger.js',
             'src/utils/api/liskService.js',
             'src/utils/ledger.js',
-            'src/utils/api/ledger.js',
-            'src/components/login/index.js',
-            'src/components/login/login.js',
             'src/utils/rawTransactionWrapper.js',
-            'src/store/middlewares/login.js',
             'src/utils/to.js',
-            'src/components/ledger/index.js',
-            'src/components/ledger/unlockWallet.js',
-            'src/components/ledger/ledgerLogin.js',
-            'src/components/ledger/addAccountCard.js',
-            'src/components/ledger/accountCard.js',
-            'src/store/reducers/liskService.js',
-            'src/actions/transactions.js',
-            'src/components/errorBoundary/index.js',
-            'src/components/request/specifyRequest.test.js',
-            'src/components/feedbackForm/**/*.js',
+            'src/components/newsFeed/index.js', // This component doesn't meet the setted tresholds for mocha but in jest
```

Comment:
Cannot this file coverage be solved just by fixing `mapDispatchToProps` as you showed us in https://blog.benestudio.co/5-ways-to-connect-redux-actions-3f56af4009c8

Reply:
No, is for a different thing, and this is a HOC component so there is no need to tested as we are removing the tests for these files